Micromouse/Maze.cpp

```diff
@@ -56,6 +56,14 @@ bool Maze::isWall(CellCoordinate pos, Direction dir) const {
     return isWall(pos.toNode() + dir);
 }
 
+bool Maze::isWall(NodeCoordinate pos, Direction dir) const {
```

Comment:
Today I learned: It is not just ctors with a single argument that need to be marked explicit, but any ctor with optional parameters that allows just a single argument also should be marked explicit.